### ISSUE TITLE
Create a rank system

### ISSUE THREAD

**Issue by kazin8old** [dydus0x14/Boobsik#1]
Please, create a simple rank system (this api may do a plus and minus on a photo)
Example: 
/boobs - show a photo
/like - like a photo
/unlike - unlike a photo

if you write /boobs again, context forward to next photo.

I love it :*

**Comment by dydus0x14** [comment]
Yes, sure. As far as I understand commands /like and /unlike rate the last photo only. Is it right?
